charts/gateway/templates/deployment.yaml

```diff
@@ -60,13 +60,35 @@ spec:
       {{- if .Values.terminationGracePeriodSeconds }}
       terminationGracePeriodSeconds: {{.Values.terminationGracePeriodSeconds }}
       {{- end }}
+      {{ if and (.Values.otk.enabled) (not .Values.database.enabled) }}
+      {{- if not .Values.initContainers  }}
```

Comment:
If .Values.initContainers is set with otk.enabled it looks like this will be an invalid deployment manifest. This will need to be tested.